src/gui/gtkentry.c

```diff
@@ -186,6 +186,8 @@ const dt_gtkentry_completion_spec *dt_gtkentry_get_default_path_compl_list()
           { "FILE_NAME", N_("$(FILE_NAME) - basename of the input image") },
           { "FILE_EXTENSION", N_("$(FILE_EXTENSION) - extension of the input image") },
           { "VERSION", N_("$(VERSION) - duplicate version") },
+          { "VERSION_IF_MULTI", N_("$(VERSION_IF_MULTI) - same as $(VERSION) only if multiple versions exist") },
```

Comment:
"same as $(VERSION) but null string if unique version" ?

Reply:
I'm not sure if 'unique version' is clear enough (since each of multiple versions could be considered unique). How about 'null string if single version exists'.

Reply:
Perfect !